### PR TITLE
Added user specific model permission to profile page

### PR DESCRIPTION
**Pull request by Lukewh**
Fixes https://github.com/juju/juju-gui/issues/2556

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7108/

**Comment by Lukewh** [comment]
## QA
Click on profile page, you should see the permissions you have on your own models and those shared with you.

**Comment by bac** [comment]
Thanks @Lukewh

File: jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js [hunk 1]
```diff
@@ -282,6 +282,16 @@ YUI.add('user-profile-model-list', function() {
       if (model.owner) {
         owner = model.owner.split('@')[0];
       }
+
+      const modelUser = model.users ? model.users.filter(user => {
+        return user.displayName === props.userInfo.profile;
+      }) : null;
+
+      let permission = 'unknown';
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js]: How would we not know what the permissions are? Can you add a comment to this effect?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7120/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**